web-registry/src/pages/ChooseCreditClass/ChooseCreditClass.tsx

```diff
@@ -21,47 +18,35 @@ const ChooseCreditClass: React.FC<React.PropsWithChildren<unknown>> = () => {
   const { projectId } = useParams();
   const { creditClassOptions, loading } = useGetCreditClassOptions();
   const { setCreditClassId } = useCreateProjectContext();
-  const { data: graphData } = useShaclGraphByUriQuery({
-    variables: {
-      uri: 'http://regen.network/ProjectPageShape',
-    },
-  });
   const [updateProject] = useUpdateProjectByIdMutation();
 
   async function handleSelection(
     creditClassId: string,
     creditClassOnChainId: string,
   ): Promise<void> {
-    if (graphData?.shaclGraphByUri?.graph) {
-      const metadata = getProjectCreateBaseData(creditClassOnChainId);
-      setCreditClassId(creditClassOnChainId);
+    const metadata = getProjectCreateBaseData(creditClassOnChainId);
+    setCreditClassId(creditClassOnChainId);
```

Comment:
i just wanted to make a note here that i would like to understand this part better, i'm having trouble visualizing what the actions in UI are that correspond to this code (choosing a credit class and why we update project. i'd like to understand this better. i'm just missing some context. not a blocker on this by any means